src/phoenix/utilities/client.py

```diff
@@ -15,6 +17,13 @@ def __init__(self, *args: Any, **kwargs: Any):
         from phoenix import __version__ as phoenix_version
 
         super().__init__(*args, **kwargs)
+
+        if env_headers := get_env_client_headers():
```

Comment:
@mikeldking the headers are now added here, which is where all the base clients used in phoenix are defined

Reply:
ah got it